server/neptune/workflows/internal/activities/terraform.go

```diff
@@ -150,10 +158,13 @@ func (t *terraformActivities) TerraformApply(ctx context.Context, request Terraf
 
 	cmd := terraform.NewSubCommand(terraform.Apply).WithInput(planFile).WithArgs(args...)
 	ch := t.TerraformClient.RunCommand(ctx, request.JobID, request.Path, cmd, request.Envs, tfVersion)
```

Comment:
JobID doesn't need to be in the request. I'll remove it in my PR. But im telling you so that in the future you remember to remove unused code.